### PR TITLE
Fixed UI of blog page

### LINKED ISSUE #412: Wrong margins/paddings in Blogs and Blog page in mobile devices.
#### Describe the bug
The margins/paddings in Blogs and Blog page in mobile devices are incorrect.

In the blogs landing there is a portion of the content missing so we need to move the content down.
Screenshot 2024-03-02 at 14 52 08

For the blog view is the opposite, there is too much space. We need to reduce it.
Screenshot 2024-03-02 at 14 52 22

Let's make sure we don't break anything for desktop and large screen devices.

### PR DESCRIPTION
**Pull request by TamannaVerma99**
**What kind of change does this PR introduce?**
A bugfix

**Issue Number:**
Resolves #412 & [#427](https://github.com/json-schema-org/website/issues/427)

**Summary**
I've made appropriate improvements to the CSS properties of the blog landing page to ensure responsiveness across all screen sizes. Additionally, I've enhanced the picture size on the landing page for better visual appeal and readability. These changes aim to create a more user-friendly and visually engaging experience for visitors accessing the blog. Also, fixed the image size to eliminate issues caused by too much padding.

### CODE REVIEW

**Comment by officeneerajsaini** [comment]
Mention #412 in your pr it will add on that issue pr , so maintainer can review or understand easily .., it can make easy to access

**Review by DhairyaMajmudar — COMMENTED** [review]
That's an amazing pr @TamannaVerma99 but can you pls. remove unwanted changes done by prettier extension so that it will be easy for maintainers to review the actual changes.

Also pls. add pr title suggesting the changes done. 

Thank You : )

**Comment by aialok** [comment]
> That's an amazing pr @TamannaVerma99 but can you pls. remove unwanted changes done by prettier extension so that it will be easy for maintainers to review the actual changes.
> 
> Also pls. add pr title suggesting the changes done.
> 
> Thank You : )

@TamannaVerma99 , If you remove prettier changes then it will throw eslint error. Make sure you have fetch the latest changes before working on issue.
Once you fetch the latest changes you will not get any additional lines change.

Thank You  : )

**Comment by TamannaVerma99** [comment]
Thank you so much for your suggestions, Really appreciate them!
However, I believe using Prettier for such Code Bases definitely counts in as one of the good coding practices. Infact, we should have Prettier & EsLint configured already in the code base to encourage them :)

**Comment by aialok** [comment]
> Thank you so much for your suggestions, Really appreciate them! However, I believe using Prettier for such Code Bases definitely counts in as one of the good coding practices. Infact, we should have Prettier & EsLint configured already in the code base to encourage them :)

Basically we do already setup the prettier and eslint. Just fetch the latest changes : )

**Review by benjagm — CHANGES_REQUESTED** [review]
Mobile looks great. For desktop there is a great improvement but I think we'll need to move the element down in the vertical axis to have them centered vertical. See image.
Screenshot 2024-03-04 at 13 47 14

We need the same height in the arrow on top and in the arrow in the bottom.

**Comment by TamannaVerma99** [comment]
okay ,I will do  the required! Thanks for the feedback :)

**Comment by TamannaVerma99** [comment]
Made the required changes @benjagm . The text is now aligned at the center for desktop view. However, it is quite improbable to do the same on all screen sizes, given the inclined nature of the image. But have tried my best to ensure appropriate text alignment accordingly on all remaining screen sizes.

**Comment by TamannaVerma99** [comment]
@benjagm can you please have a look on it?

File: tailwind.config.js [hunk 1]
```diff
@@ -4,6 +4,24 @@ module.exports = {
     './components/**/*.{js,ts,jsx,tsx,md}',
   ],
   theme: {
+    screens: {
```
Comment (benjagm) [inline tailwind.config.js:7]: Is this code necessary?

**Review by benjagm — APPROVED** [review]
Looks great. Just left a comment regarding editing tailwind.config.js. I am not sure if the new code there is necessary.

**Comment by TamannaVerma99** [comment]
@benjagm in order to add custom screen sizes in Tailwind, its necessary to specify the default screen sizes as well to prevent overwriting them. So, needed to add them as well along with the new custom size (ab1 :'890px')

**Comment by aialok** [comment]
Hey @TamannaVerma99, it seems like you haven't formatted the code with Prettier. Now, when I fetch the latest changes, I'm getting ESLint errors for the file you've edited. Did you bypass the Husky pre-commit checks? 

I think you should format and check for ESLint errors before pushing your code. Otherwise, it could create problems for other contributors who are working on the project.

Screenshot from 2024-03-05 21-27-27
Screenshot from 2024-03-05 21-27-06

Well, we've recently implemented a PR workflow where we check formatting, ESLint, and type checking. This should prevent this type of issues in the future.

@benjagm, this needs to be fixed as soon as possible. You can either revert this PR or fix the issue from your end.

Thank you : )

**Comment by benjagm** [comment]
Fixed in production. Thanks everyone.

**Pull request merged**